### ISSUE TITLE
[DevOps] ESECC documentation for outgoing Lower Environment connection to Vista API

### ISSUE THREAD

**Issue by olivereri** [department-of-veterans-affairs/va.gov-team#88645]
## Description
This needs to be researched and completed.

## Tasks
- [ ] [DevOps] Determine what needs to be documented in order to support this functionality.
- [ ] [DevOps] Complete this documentation or open up followup ticket(s) to address necessary details.

## Acceptance Criteria
- [ ] Documentation is completed and submitted, or
- [ ] Followup ticket(s) opened to address complications.

## References
ESECC Documentation:
https://github.com/department-of-veterans-affairs/checkin-devops/blob/main/docs/esecc-requests.md

Issue history:
https://github.com/department-of-veterans-affairs/chip/issues/1125
https://github.com/department-of-veterans-affairs/chip/issues/1129

**Comment by olivereri** [comment]
In the ESECC request documentation found in the issue description there is a list of open Requests For Change (RFC).  Requesting and in scope change for those existing RFCs appears to be the most expedient way to open ports for the `vaec-cie` account. [RFC-008811](https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fesecc.va.gov%2Fcgweb%2FMainUI%2FMainFrame.aspx%3FID%3D11803%26moduleName%3DRFC&data=05%7C01%7C%7C17f781f985174487b11808da9b2dc44e%7Ce95f1b23abaf45ee821db7ab251ab3bf%7C0%7C0%7C637992918108148859%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=sCLx%2ByH0oS%2FZzeMjQ8dJChhukYI2syIn5DPX1cgk%2F5o%3D&reserved=0) for the dev env is currently in the "pending approval" stage of another in scope change. Once that is complete I will initiate an in scope change with a completed BPE worksheet.

**Comment by olivereri** [comment]
RFC-008811's previous change is still moving along towards implementation. I had a thought that since the connection ID was changing I should submit an entirely new BPE. However, reviewing other BPE spreadsheets for that RFC there are different connection IDs included in "In Scope" changes to support VeText.  Confirms my decision to use RFC-008811 and wait until I can submit a new in-scope change.

**Comment by olivereri** [comment]
I think I went overboard with this ticket and started driving straight toward implementation.  The existing ESECC documentation linked in the issue body was sufficient to push me far along in to implementation. I should have closed this after finding the existing ESECC docs in `checkin-devops`
